### PR TITLE
OOP school library

### PR DESCRIPTION
**Pull request by eerapheal**
In this project, I initialise the school library app. In this initial step, I implement the classes to represent students and teachers.

### CODE REVIEW

File: student_s.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
```
Comment (OLIPLICHE) [inline student_s.rb]: - Please `name` and `parent_permission` are optional and have default values of "Unknown" and "true". [Refer project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md)

File: teacher_t.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(name, age, parent_permission, specialization)
```
Comment (OLIPLICHE) [inline teacher_t.rb]: - Please `name` and `parent_permission` are optional and have default values of "Unknown" and "true". [Refer project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md)

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @eerapheal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @eerapheal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**